config/config.go

```diff
@@ -244,6 +234,28 @@ func (s *SourceSubnet) Validate() error {
 		}
 	}
 
+	// Validate the allowed destinations
+	for _, chainIDs := range s.SupportedDestinations {
+		if _, err := ids.FromString(chainIDs); err != nil {
+			return fmt.Errorf("invalid chainID in source subnet configuration. Provided ID: %s", chainIDs)
+		}
+	}
```

Comment:
Is this loop needed? Looks like we have the same check in the iteration below already.

Reply:
Nope, it's redundant. Removed.